### DISCUSSION TITLE
Cloud Functions Implementation

### DISCUSSION THREAD

**Discussion by lmick002** [supabase/supabase#888] [Questions]
Is there a current way to implement cloud functions like triggers on database changes currently? I know the feature is being worked on now but is there any workaround until the actual feature is released? Thanks!

**Comment by steve-chavez** [comment]
You can use regular SQL trigger for now([tutorial](https://www.postgresqltutorial.com/creating-first-trigger-postgresql/)).

If you need an http call on these triggers, you can use pgsql-http. Check https://supabase.io/blog/2021/03/05/postgres-as-a-cron-server#postgres--webhooks
